### ISSUE TITLE
Levitating buff doesn't allow user to walk over piranha

### ISSUE THREAD

**Issue by jamessan** [watabou/pixel-dungeon#71]
With an invisibility buff, I went into one of the piranha rooms and the piranhas then moved to block the door.  No problem, I thought, I can just use a Potion of Levitation to walk out of the room.  However, the game won't let me navigate past the fish.

**Comment by lighthouse64** [comment]
It won't let you pass through any enemies.  The potion of levitation only protects you from trampling on tiles.

**Comment by markozajc** [comment]
The game will usually give you Potion of Invisibility to get past the piranhas in piranha rooms' floors.

**Comment by jamessan** [comment]
> The game will usually give you Potion of Invisibility to get past the piranhas in piranha rooms' floors.

Right, that's what I was mentioning in my original post.  However, I didn't have one and figured levitation would be sufficient.
